development/db/structuredConditionals.rst

```diff
@@ -0,0 +1,504 @@
+===========================
+The boolean structure system
+===========================
```

Comment:
That's definitely the ***** part of RST ;)

The line must have exactly has much characters as the text